website/docs/utilities/dialog-primitive/partials/guidelines/guidelines.md

```diff
@@ -0,0 +1,23 @@
+## Usage
+
+The DialogPrimitive sub-components are not intended to be used independently of each other or in isolation. If you are looking to create your own dialog-based component, use the `DialogPrimitive::Wrapper` as your foundation.
+
+A common example of this is a custom dialog-based element such as a SplitWindow/Pane/Panel which allows the user to interact with both the parent page and the content within the secondary element (often referred to as a "non-modal" dialog).
+
```

Comment:
@jorytindall The reference to the "the secondary element" makes this confusing to understand (I assume it means the dialog-based component?).

Reply:
How about:

A common example of this is a non-modal dialog such as a SplitWindow or Panel which allows the user to interact with both the main page content and the content within the dialog.

Reply:
Yep, that's much cleaner, thanks for the recommendation